packages/dream-html/dream-html.3.9.5/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "HTML generator eDSL for Dream"
+description:
+  "Write HTML directly in your OCaml source files with editor support."
+maintainer: ["Yawar Amin <yawar.amin@gmail.com>"]
+authors: ["Yawar Amin <yawar.amin@gmail.com>"]
+license: "GPL-3.0-or-later"
+tags: ["org:yawaramin"]
+homepage: "https://github.com/yawaramin/dream-html"
+doc: "https://yawaramin.github.io/dream-html/"
+bug-reports: "https://github.com/yawaramin/dream-html/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "ocaml" {>= "4.14.0"}
+  "ppxlib" {>= "0.33.0" & < "1.0.0"}
+  "pure-html" {= version}
+  "dream" {>= "1.0.0~alpha7"}
```

Comment:
The only relevant failure in CI is the lowe bounds check failing on

```
[ERROR] The compilation of dream.1.0.0~alpha7 failed at "dune build -p dream -j 39".

#=== ERROR while compiling dream.1.0.0~alpha7 =================================#
# context              2.4.0~alpha1~dev | linux/x86_64 | ocaml-base-compiler.5.3.0~beta1 | file:///home/opam/opam-repository
# path                 ~/.opam/5.3~beta1/.opam-switch/build/dream.1.0.0~alpha7
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p dream -j 39
# exit-code            1
# env-file             ~/.opam/log/dream-7-e90cf6.env
# output-file          ~/.opam/log/dream-7-e90cf6.out
### output ###
# File "src/http/dune", line 21, characters 2-32:
# 21 |   dream-httpaf.dream-websocketaf
#        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Library "dream-httpaf.dream-websocketaf" not found.
# -> required by library "dream.http" in _build/default/src/http
# -> required by _build/default/META.dream
# -> required by _build/install/default/lib/dream/META
# -> required by _build/default/dream.install
# -> required by alias install
```

Since you know dream quite well, iiuc, do you know if this is a packaging error on the side of dream? Could it be avoided for your package perhaps just by raising the lower bound to dream alpha8?

Reply:
Hi, I think this is because of #27062. Sure, we can try bumping the lower bound.